docs/reference/release-notes/highlights-6.7.0.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[release-highlights-6.7.0]]
+== 6.7.0 release highlights
+++++
+<titleabbrev>6.7.0</titleabbrev>
+++++
+
+See also <<release-notes-6.7.0,{es} 6.7.0 release notes>>.
+
+[float]
+=== Cross-cluster replication
+
+We have introduced cross-cluster replication in 6.5.0 as a beta feature and it
+is now mature enough that we make it generally available. Cross-cluster
```

Comment:
We introduced cross-cluster replication in 6.5.0 as a beta feature, it
is now mature enough to declare the feature GA (general availability).